src/JustSaying.Extensions.DependencyInjection.StructureMap/JustSayingRegistry.cs

```diff
@@ -36,6 +37,7 @@ public JustSayingRegistry()
         For<IPublishBatchConfiguration>().Use<MessagingConfig>(context => context.GetInstance<MessagingConfig>()).Singleton();
         For<IMessageMonitor>().Use<NullOpMessageMonitor>().Singleton();
         For<IMessageBodySerializationFactory>().Use<NewtonsoftSerializationFactory>().Singleton();
+        For<JsonSerializerSettings>().Use(() => new JsonSerializerSettings()).Singleton();
```

Comment:
In new minor versions of `Newtonsoft.Json`, there are multiple constructors for `JsonSerializerSettings`, so this makes it explicit.